client/src/views/TripHomeView.vue

```diff
@@ -73,90 +60,41 @@
-          <span class="text-sm font-normal text-gray-500">11:46</span>
-        </div>
-        <p class="text-sm font-normal py-2.5 text-gray-900">
-          Les gjennom fjellvettreglene en siste gang før vi drar ut på tur:
-        </p>
-        <p class="text-sm font-normal pb-2.5 text-gray-900">
-          <a
-            href="https://www.dnt.no/fjellvettreglene/"
-            class="text-blue-700 underline hover:no-underline font-medium break-all"
-            >https://www.dnt.no/fjellvettreglene/</a
-          >
-          <img
-            src="https://www.dnt.no/globalassets/fotoware/2023/12/0323massivfoto_mdalseg_16.jpg?width=1920&format=webp"
-            class="rounded-lg my-2 max-w-md"
-          />
-        </p>
-      </div>
-    </div>
-    <div class="flex items-start gap-2.5">
-      <img
-        src="https://cdn.myanimelist.net/r/42x62/images/characters/2/366169.webp?s=afe9005b70cf1d0193fcd769e020a317"
-        class="object-cover rounded-full h-10 w-10 my-3"
-      />
-      <div
-        class="flex flex-col w-full my-3 mr-5 leading-1.5 p-4 border-gray-200 bg-gray-100 rounded-e-xl rounded-es-xl"
-      >
-        <div class="flex items-center space-x-2 rtl:space-x-reverse">
-          <span class="text-sm font-semibold text-gray-900">Nadeshiko Kagamihara</span>
-          <span class="text-sm font-normal text-gray-500">11:50</span>
-        </div>
-        <p class="text-sm font-normal py-2.5 text-gray-900">
-          Fjellvettreglene sjekket! Jeg har også pakket ekstra snacks for alle. 🍙 Kan ikke vente
-          med å se stjernene fra Rustfjellhei! ✨
-        </p>
-      </div>
-    </div>
-    <div class="flex items-start gap-2.5">
-      <AnonymousUserIcon />
-      <div
-        class="flex flex-col w-full my-3 mr-5 leading-1.5 p-4 border-gray-200 bg-gray-100 rounded-e-xl rounded-es-xl"
-      >
-        <div class="flex items-center space-x-2 rtl:space-x-reverse">
-          <span class="text-sm font-semibold text-gray-900">Aoi Inuyama</span>
-          <span class="text-sm font-normal text-gray-500">11:55</span>
-        </div>
-        <p class="text-sm font-normal py-2.5 text-gray-900">
-          Husk å ta med varme klær, folkens! Det blir kjølig om natten. 🧣 Og Nadeshiko, ikke glem
-          teltet denne gangen! 😅
-        </p>
+    <form v-if="isParticipant" @submit.prevent="createAttachment">
+      <div class="flex flex-col items-left">
+        <textarea v-model="attachment.text"
+          class="w-full px-3 py-2 border border-gray-300 rounded-lg shadow-sm focus:ring-blue-500 focus:border-blue-500 sm:text-sm text-gray-900"
+          placeholder="Skriv et innlegg..." />
+        <input type="file" v-on:change="uploadFile" />
+        <button type="submit"
+          class="inline-flex items-center justify-center px-3 py-2 mt-2 text-xs font-medium text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:ring-blue-300">
+          Publiser
+        </button>
```

Comment:
Designet her var litt ... manglende?

Heldigvis trenger du ikke være en kreativ sjel for å lage noe som ser sånn halvveis ut, vi bruker allerede flowbyte til det meste, og de har en [comment box](https://flowbite.com/docs/forms/textarea/#comment-box) og en [chatroom input](https://flowbite.com/docs/forms/textarea/#comment-box). Kan du ikke mekke til noe fra en av disse? Jeg foretrekker kanskje noe som ligner mest på "comment box".

Reply:
> lett å lage noe som ser halvveis ut
> <lenke til html-eksempel med 229 i kolonnebredde> (321 hvis man inkluderer svg paths)

Anyway, jeg har byttet om litt nå, inspirert av "comment box", men gadd ikke bytte bort fra file-input

Reply:
Eeeeehhhhhh good enough